net-voip/ring-daemon/ring-daemon-99999999.ebuild

```diff
@@ -125,13 +126,18 @@ src_configure() {
 		rm -r src/pjproject
 	fi
 
+	#if system library is installed, then bundled will be ignored even with !system-
```

Comment:
This behavior assembly system took time to figure out why it is ignored  -system-* USE and used system libs instead of bundled. But then i removed system libs and it started to build bundled libs.